targets/hasura/migrations/1613478848457_add_audit/up.sql

```diff
@@ -0,0 +1,35 @@
+--
+-- Audit the following table using audit triggers
+--
+
+select audit.audit_table('auth.refresh_tokens');
+select audit.audit_table('auth.user_roles');
+select audit.audit_table('auth.users');
+select audit.audit_table('alert_notes');
+select audit.audit_table('alert_status');
+select audit.audit_table('alerts');
+select audit.audit_table('document_relations');
+select audit.audit_table('glossary');
+select audit.audit_table('kali_blocks');
+select audit.audit_table('package_version');
+select audit.audit_table('roles');
+select audit.audit_table('sources');
```

Comment:
Or perhaps Hasura can handle this kind of change?

Reply:
+1 for documentation 
audit a table should be done by hand. since audit info are heavy we should decide which table we need to track (and at wich level)